packages/block-editor/src/components/block-preview/README.md

```diff
@@ -29,3 +29,26 @@ A block instance (object) or a blocks array you would like to render a preview.
 Width of the preview container in pixels. Controls at what size the blocks will be rendered inside the preview.
 
 `viewportWidth` can be used to simulate how blocks look on different device sizes or to make sure make sure multiple previews will be rendered with the same scale, regardless of their content.
+
+### `__experimentalOnReady`
+* **Type** `Function`
+* **Default:** `noop`
+
+Use this callback as an opportunity to know when the preview is ready. The callback will pass, if available:
+
+* `scale`: the scale factor
+* `position`: offsets position (x, y)
+* `previewContainerRef`: DOM element reference
+* `styles`: Online styles applied to the preview container
```

Comment:
Did you mean "Inline styles applied.."? 
Should the property be called inlineStyles instead of styles to be more specific?